pages/api/funding/balanceOnDate.ts

```diff
@@ -44,10 +50,15 @@ export default async function handler(
     },
   });
 
+  const allocated: { [key: string]: BigNumber } = Object.keys(
```

Comment:
should this be a const?

Reply:
yeah, as we only replace the values under specific keys not the object itself.